docs/setup/vagrant.rst

```diff
@@ -2,6 +2,8 @@
 Setting up Vagrant
 ******************
 
+**The following commands should be executed in your host computer's console. You may want to create a new directory for easy access in the future.**
```

Comment:
```suggestion
**The following commands should be executed in your host computer's console.**
```

Cloning the Git repo will already create a new directory. Instructing users to create one is redundant.